labs/basic-template/basic-template.json

```diff
@@ -1,12 +1,14 @@
 {
   "labId": "basic-template",
-  "title": "Template For A Lab In DevNet Learning Labs",
+  "title": "Template for a lab in DevNet Learning Labs",

```

Comment:
Oh maybe Learning Labs is capital, but "lab" doesn't have to be? Up to you as long as it's consistent.

Reply:
Oh, I see, this can be the generic template title. Got it! I'll merge.